source/client-side-encryption/client-side-encryption.rst

```diff
@@ -694,7 +694,7 @@ If the collection namespace has an associated ``encryptedFields``, then do the f
 - Create the the index ``{"__safeContent__": 1}`` on collection ``collectionName``.
 
 Drop Collection Helper
-``````````````````````
+^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
My mistake. I broke this in #1232 when looking at the previous sections for guidance. Backticks are the heading syntax used _after_ carets.

Reply:
We made https://github.com/mongodb/mongoid/blob/master/docs/nesting-levels.txt in Mongoid to act as a quick reference for the correct heading syntax to use.